deploy/olm-catalog/ibm-block-csi-operator/rh_bundle/ibm-block-csi-operator-bundle_1_4_0/1.4.0/metadata/annotations.yaml

```diff
@@ -0,0 +1,7 @@
+annotations:
+  operators.operatorframework.io.bundle.channel.default.v1: stable
+  operators.operatorframework.io.bundle.channels.v1: stable
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: ibm-block-csi-operator
```

Comment:
could we change the order of these to match the dockerfile labels order? just to ease future comparisons

Reply:
done - Dockerfile files have been updated